library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608Parser.java

```diff
@@ -153,6 +156,12 @@
         continue;
       }
 
+      // Ignore repeated control characters.
+      if (ccData1 == previousCtrl1 && ccData2 == previousCtrl2) {
```

Comment:
I am trying to implement the missing part, that is why I read about the commands, but I have not dealt with duplicates yet. I was arguing for (b), as in the renderer the second incoming command seems simply overriding the first (I modified the pop-up type). But your notes are valid, TBoshoven might update the change set.

First:
It seems that "previousCtrl1" is only set when "ccData1 < 0x20", but lower values can still be present and dropped.

Second:
When we exit the loop because of the condition "ccData1 == previousCtrl1 && ccData2 == previousCtrl2", we might want to reset previousCtrl1 and previousCtrl2 to 0, so the 3rd exact same incoming command message wont be dropped (but the 4th again etc.)